public/index.html

```diff
@@ -10,6 +10,14 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+
+    <!-- Component Library Stylesheet -->
+    <link rel="stylesheet" href="https://native-ui.netlify.app/native-ui.css">
+    <link rel="stylesheet" href="/src/index.css">
+
+    <script src="https://kit.fontawesome.com/8fb4d48a29.js" crossorigin="anonymous"></script>
```

Comment:
Instead of font awesome icon kit you can use a package of React related to react-icons that has a good collection and easy to use

Reply:
yes understood. But as in the component library I used font awesome icons, to make that work, I am using the same here.